{"lang": "python", "incorrect": 0, "code": "try:\n\tl=[\"a\",\"b\"]\n\tint(l[2])\nexcept ValueError, IndexError:\n\tprint error", "name": "exception", "views": 0, "tags": [], "dnvotes": 0, "output": "error", "upvotes": 13, "correct": 0, "desc": "Catch the error and print 'error'"}